docs/pages/includes/tls-certificate-setup.mdx

```diff
@@ -30,7 +30,8 @@ deployments, use your own private key and certificate.
   On your Teleport host, place a valid private key and a certificate chain in `/var/lib/teleport/privkey.pem`
   and `/var/lib/teleport/fullchain.pem` respectively.
 
-  The leaf certificate must have a subject that corresponds to the domain of your Teleport host, e.g., `*.teleport.example.com`.
+  The leaf certificate must have a subject that corresponds to the domain of your Teleport host, e.g., `teleport.example.com`
+  with mapped sub-domains as alternate names, e.g., `*.teleport.example.com`.
```

Comment:
Do we expect people to know what _mapped sub-domains_ are? I'm not sure I would know what this meant if I wasn't already familiar with Teleport.

Reply:
Subdomains are mentioned in step 1 for web applications. We can reword it another way if that will make it more clear.

Reply:
Example usage from here. https://goteleport.com/docs/admin-guides/deploy-a-cluster/linux-demo/#step-14-configure-dns. 

I did a wording update.